src/server_manager/infrastructure/data_formatting.spec.ts

```diff
@@ -0,0 +1,48 @@
+/*
+  Copyright 2020 The Outline Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import * as i18n from './data_formatting';
+
+// Jasmine with Node only supports 'en'
+if (Intl.NumberFormat.supportedLocalesOf(['en', 'fr', 'fa', 'es', 'ru']).length === 5) {
+  describe('getFormattedDataAmountParts', () => {
+    it('extracts the unit string and value separately', () => {
+      const english = i18n.getFormattedDataAmountParts(0, 'en');
```

Comment:
Could you add an example where the value is 2 bytes?

Reply:
Done in 8c1a131cb12e6b52de6008df1eaf5526011fa487